README.md

```diff
@@ -1,16 +1,17 @@
 # Percona Monitoring and Management 2.x
+
 [![CI](https://github.com/percona/pmm/actions/workflows/common.yml/badge.svg)](https://github.com/percona/pmm/actions/workflows/common.yml)
 [![CLA assistant](https://cla-assistant.percona.com/readme/badge/percona/pmm)](https://cla-assistant.percona.com/percona/pmm)
 [![Code coverage](https://codecov.io/gh/percona/pmm/branch/main/graph/badge.svg)](https://codecov.io/gh/percona/pmm)
 [![Go Report Card](https://goreportcard.com/badge/github.com/percona/pmm)](https://goreportcard.com/report/github.com/percona/pmm)
 [![Discord](https://img.shields.io/discord/808660945513611334?label=Discord&logo=Discord&style=flat)](https://per.co.na/discord)
 [![Forum](https://img.shields.io/badge/Forum-join-brightgreen)](https://forums.percona.com/)
 
-![PMM](https://www.percona.com/sites/default/files/pmm-logo.png)
+![PMM](https://www.percona.com/wp-content/uploads/2023/01/logo.svg)
```

Comment:
is it a PMM Logo or Percona one?

Reply:
This is Percona logo, we need here PMM logo

Reply:
Right, I didn't realize. Do we have one? Asking the PM...

Reply:
Added PMM product logo

Reply:
That's what the UX experts recommend us to use.

Reply:
Probably they thought that GH has only light theme

Reply:
One more version - seems to be better now :)

<img width="1639" alt="Screenshot 2023-02-16 at 13 13 15" src="https://user-images.githubusercontent.com/81549/219337497-89b7f019-0bdf-424f-bd98-0c91e4025f85.png">

Reply:
IMHO, the one with white logo looks better